downloader/src/main/kotlin/vcs/Git.kt

```diff
@@ -60,10 +63,9 @@ class Git : VersionControlSystem(), CommandLineTool {
         init {
             installAuthenticatorAndProxySelector()
 
-            // While the OrtAuthenticator already provides .netrc parsing, JGit will complain with a confusing "no
-            // CredentialsProvider registered" message if authentication fails. Avoid that by installing JGit's own
-            // .netrc credential provider in addition to ORT's authenticator.
-            NetRCCredentialsProvider.install()
+            // Make sure that JGit uses the exact same authentication information as ORT itself. This addresses
+            // discrepancies in the way .netrc files are interpreted between JGit's and ORT's implementation.
```

Comment:
Just out of curiosity: So did you verify by looking at the code again that the implementations do parse the `.netrc` differently, or was this just an observation from trying this out? In other words, did you find a bug in JGit's `.netrc` parsing regarding default entries, and would you be able to contribute a fix to JGit?

Reply:
I did not look at the code. It was an observation based on my experiments. So I can neither claim that there is a bug nor can I provide a fix.